brizo/routes.py

```diff
@@ -217,38 +229,191 @@ def consume():
         logger.info(f'Done processing consume request for asset {did}, agreementId {agreement_id},'
                     f' url {download_url}')
         return build_download_response(request, requests_session, url, download_url, content_type)
+
+    except InvalidSignatureError as e:
+        msg = f'Consumer signature failed verification: {e}'
+        logger.error(msg, exc_info=1)
+        return msg, 401
+
     except (ValueError, Exception) as e:
         logger.error(f'Error- {str(e)}', exc_info=1)
         return f'Error : {str(e)}', 500
 
 
-@services.route('/exec', methods=['POST'])
-def execute_compute_job():
-    """Call the execution of a workflow.
+@services.route('/compute', methods=['DELETE'])
+def compute_delete_job():
+    """Deletes a workflow.
 
     ---
     tags:
       - services
     consumes:
       - application/json
     parameters:
+      - name: signature
+        in: query
+        description: Signature of the documentId to verify that the consumer has rights to download the asset.
+        type: string
+      - name: serviceAgreementId
+        in: query
+        description: The ID of the service agreement.
+        required: true
+        type: string
       - name: consumerAddress
```

Comment:
parameter should not be named `consumerAddress` if it is in fact the owner address